fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HMSTransaction.java

```diff
@@ -558,10 +564,11 @@ private DeleteRecursivelyResult recursiveDeleteFiles(Path directory, boolean del
             return new DeleteRecursivelyResult(false, notDeletedEligibleItems.build());
         }
 
-        return doRecursiveDeleteFiles(directory, deleteEmptyDir);
+        String queryId = DebugUtil.printId(ConnectContext.get().queryId());
```

Comment:
I suggest to save query id when generating this HiveTransaction.
Because `ConnectContext.get()` is a thread local var, it may be null if we run this transaction in another thread